docs/Tutorial_Consensus_Motif.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>I've uploaded cleaner CSV files to Zenodo. Can you please replace this cell with:</p><p> </p><pre class="ql-syntax" spellcheck="false">animals = ['python', 'hippo', 'red_flying_fox', 'alpaca']
data = {}
for animal in animals:
&nbsp; &nbsp; data[animal] = pd.read_csv(f"https://zenodo.org/record/4289120/files/{animal}.csv?download=1").iloc[:,0].values
&nbsp; &nbsp;&nbsp;
colors = {'python': 'tab:blue', 'hippo': 'tab:green', 'red_flying_fox': 'tab:purple', 'alpaca': 'tab:red'}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Thanks!</p>